spanner/spanner_snippets/snippet.go

```diff
@@ -88,6 +90,11 @@ var (
 		"querywithint":                queryWithInt,
 		"querywithstring":             queryWithString,
 		"querywithtimestampparameter": queryWithTimestampParameter,
+		"querywithqueryoptions":       queryWithQueryOptions,
+	}
+
+	newClientCommands = map[string]newClientCommand{
```

Comment:
s/newClientCommands/newClientCommand/g

Reply:
Or with the s, but needs to be consistent. :)

Reply:
`newClientCommands` is a collection (mapping from string to newClientCommand). If you look at other declarations in this file, this is consistent. 

https://github.com/GoogleCloudPlatform/golang-samples/blob/1bd6ff8a213e97ad19565b4a636a4a82405c2cc5/spanner/spanner_snippets/snippet.go#L43

https://github.com/GoogleCloudPlatform/golang-samples/blob/1bd6ff8a213e97ad19565b4a636a4a82405c2cc5/spanner/spanner_snippets/snippet.go#L100